src/spark/primitives.h

```diff
@@ -105,6 +107,7 @@ class CSparkOutputTx
     inline void SerializationOp(Stream& s, Operation ser_action) {
         READWRITE(address);
         READWRITE(amount);
+        READWRITE(memo);
```

Comment:
you need a check here, to prevent issues in case you have an old type data without this field .